tests/api_tests/nodes/test_serializers.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+from urlparse import urlparse
+
+from nose.tools import *  # flake8: noqa
+from dateutil.parser import parse as parse_date
+
+from tests.base import DbTestCase, ApiTestCase, assert_datetime_equal
+from tests.utils import make_drf_request
+from tests.factories import UserFactory, NodeFactory, RegistrationFactory, ProjectFactory
+
+from framework.auth import Auth
+from api.nodes.serializers import NodeSerializer, NodeRegistrationSerializer
+from api.base.settings.defaults import API_BASE
+
+
+class TestNodeSerializer(DbTestCase):
```

Comment:
Makes sense. I just figured the form is what the schema is looking at and the last thing that **should** be touching the form is the serializer. Runscope tests are great but if something is changing the format of the json between the serializer and the wire we've got problems.